week1/project/ecommerce/src/components/Products.jsx

```diff
@@ -0,0 +1,20 @@
+import './Products.css';
+import Product from './Product'
+
+  
+  function Products({products, state}) {
+
+    const filter = state.isAll ? '' : state.selectedCategory.slice(6)
```

Comment:
naming is critical .. I think this variable should be `categoryName` or something similar.
The first time I read the variable name I thought it's filtered items, which confused me

Reply:
fixed